### PR TITLE
Fixed weights for all real firearms.

### LINKED ISSUE #34830: 3/4ths of all real guns are heavily underweight.
**Is your feature request related to a problem? Please describe.**  
Most firearms do not currently have accurate weights applied to them. While I do not yet have a comprehensive list made up, the theme seems to be ~3/4 of all real, brand-name guns (i.e. not RivTech or homemade) are underweight by ~10-20%, with some being up to half their real-world weight. Some are also overweight, but this is rarer.
While this does not necessarily break the balance of the game, it does completely ruin the balance of firearms with respect to each other, seeing as how weight directly affects aim time, recoil, etc.

**Describe the solution you'd like**  
Accurate weights should eventually be applied to all real firearms, especially the most egregious offenders. *cough cough* 416 *cough*

**Additional context**  
Google Sheets spreadsheet with all real firearms and their current modeled weights/real-world weights.
https://docs.google.com/spreadsheets/d/1amMgy_2Yq13aScDdUp7b8HSUS6mZLT9Ddjxoc9k3ruI/edit?usp=sharing
Please feel free to make comments.
UPDATE: Table now sort-able by percent difference between real and in-game weight, so you can easily see the worst of it.

### PR DESCRIPTION
**Pull request by Warriorcat49**
Reference issue #34830.

#### Summary

SUMMARY: Balance "Fixed weights for all real firearms."
#### Purpose of change

Fixes #34830 - 3/4ths of all real guns are heavily underweight.
#### Describe the solution

Accurate weights have been applied to all real firearms, with exception to Savage 111F and Taurus Pro .38 (These firearms do not exist as currently modeled and cannot have accurate weights ascribed to them until otherwise changed). Firearms within 1% of real weight have been left because they're probably accurate enough as they are.

#### Additional context

Google Sheets spreadsheet with all real firearms and their current modeled weights/real-world weights.
https://docs.google.com/spreadsheets/d/1amMgy_2Yq13aScDdUp7b8HSUS6mZLT9Ddjxoc9k3ruI/edit?usp=sharing

### CODE REVIEW

File: data/json/items/gun/22.json [hunk 1]
```diff
@@ -333,8 +334,8 @@
     "looks_like": "glock_17",
     "type": "GUN",
     "name": "Walther P22",
-    "description": "The Walther P22 is a blowback operated semi-automatic pistol.  It is about half the size of most Walthers.  It is made mostly with plastic, with the slide and key components being made of die-cast zinc alloy",
-    "weight": "480 g",
+    "description": "The Walther P22 is a blowback operated semi-automatic pistol.  It is about half the size of most Walthers. It is made mostly with plastic, with the slide and key components being made of die-cast zinc alloy",
```
Comment (ZhilkinSerg) [inline data/json/items/gun/22.json]:
```suggestion
    "description": "The Walther P22 is a blowback operated semi-automatic pistol.  It is about half the size of most Walthers.  It is made mostly with plastic, with the slide and key components being made of die-cast zinc alloy",
```

File: data/json/items/gun/762R.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     "name": "Mosin-Nagant M44",
     "name_plural": "Mosin-Nagant M44",
     "description": "The Model 44 is the carbine sized variant of the standard 1891/30.  It has a shorter barrel, therefore it is weaker than the 1891/30, but easier to carry.",
+    "weight": "4100 g",
     "price": 23000,
     "relative": { "weight": -500, "volume": -2, "range": -6, "ranged_damage": -3, "dispersion": 60, "barrel_length": -2 }
```
Comment (kevingranade) [inline data/json/items/gun/762R.json]:
```suggestion
    "relative": { "volume": -2, "range": -6, "ranged_damage": -3, "dispersion": 60, "barrel_length": -2 }
```

Comment (ZhilkinSerg) [reply data/json/items/gun/762R.json]: One whitespace off.

Comment (anothersimulacrum) [reply data/json/items/gun/762R.json]:
```suggestion
    "relative": { "volume": -2, "range": -6, "ranged_damage": -3, "dispersion": 60, "barrel_length": -2 }
```

**Pull request merged**